c2corg_ui/static/js/apiservice.js

```diff
@@ -390,17 +390,18 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
  * @param {undefined | string} token
  * @param {string} lang
  * @param {boolean} isProfile
+ * @param {boolean} personalFeed
  * @return {!angular.$q.Promise<!angular.$http.Response>}
  */
-app.Api.prototype.readFeed = function(token, lang, isProfile) {
+app.Api.prototype.readFeed = function(token, lang, isProfile, personalFeed) {
```

Comment:
What about a more explicit var name for a boolean such as ``isPersonal``?
More consistent with ``isProfile``.